doc/source/serve/tutorials/text-classification.md

```diff
@@ -0,0 +1,47 @@
+(serve-text-classification-tutorial)=
```

Comment:
Could you apply the same wording suggestions I made for `object-detection.md` to this file also?